In [1]:
import pandas as pd

In [3]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [4]:
path = "/content/drive/MyDrive/Intermediate Machine Learning/winequality-red.csv"
df = pd.read_csv(path)
df.head()

fixed acidity  volatile acidity  citric acid  residual sugar  chlorides  \
0            7.4              0.70         0.00             1.9      0.076   
1            7.8              0.88         0.00             2.6      0.098   
2            7.8              0.76         0.04             2.3      0.092   
3           11.2              0.28         0.56             1.9      0.075   
4            7.4              0.70         0.00             1.9      0.076   

   free sulfur dioxide  total sulfur dioxide  density    pH  sulphates  \
0                 11.0                  34.0   0.9978  3.51       0.56   
1                 25.0                  67.0   0.9968  3.20       0.68   
2                 15.0                  54.0   0.9970  3.26       0.65   
3                 17.0                  60.0   0.9980  3.16       0.58   
4                 11.0                  34.0   0.9978  3.51       0.56   

   alcohol  quality  
0      9.4        5  
1      9.8        5  
2      9.8        5  
3      9.8        6  
4      9.4        5

In [5]:
df.shape

(1599, 12)

This data set comes from Kaggle. It is a collection of wines, 1599 to be exact, the readings and makeups of the wines. The target is to predict the quality of the wine from things like sulphates, sugar and carbon dioxide. Each row represents a single wine. This is a regression project. There are 1599 wines in the dataset and 12 columns or features. the data per the literature on the kaggle website indicates that the data itself is quite clean, of course I will check myself. It also mentions that there are lots of normal wines and very few poor or exceptional wines. There will be lots of opportunities to explore weighting the dataset. 

In [7]:
path = "/content/drive/MyDrive/Intermediate Machine Learning/diabetes.csv"
df = pd.read_csv(path)
df.head()

Pregnancies  Glucose  BloodPressure  SkinThickness  Insulin   BMI  \
0            6      148             72             35        0  33.6   
1            1       85             66             29        0  26.6   
2            8      183             64              0        0  23.3   
3            1       89             66             23       94  28.1   
4            0      137             40             35      168  43.1   

   DiabetesPedigreeFunction  Age  Outcome  
0                     0.627   50        1  
1                     0.351   31        0  
2                     0.672   32        1  
3                     0.167   21        0  
4                     2.288   33        1

In [9]:
df.shape

(768, 9)

this data set comes from Kaggle and represnets a group of Pima Indian women over the age of 21. the goal of the dataset is to determine if the women have diabetes or not. Each row represents a person. It it classification problem. 
the data has 9 features and 768 rows. The data according to kaggle is very clean. That said I will try to clean the data and look for missing values.